doc/developer/release-checklist.md

```diff
@@ -18,7 +25,10 @@ production readiness.
   v0.2.3 would be called v0.2.3-rc1.
 
 - [ ] Update the version field in [`src/materialized/Cargo.toml`](../../src/materialized/Cargo.toml)
-      and [`LICENSE`](/LICENSE) and commit that change.
+  and [`LICENSE`](/LICENSE) and commit that change.
```

Comment:
Can we expand out the LICENSE and Cargo.toml changes into two sub checkboxes since they'll be easier to forget with the diff hidden now?

Reply:
done, makes sense.